common/src/main/java/io/eventuate/examples/tram/ordersandcustomers/commondomain/MoneyDTO.java

```diff
@@ -0,0 +1,61 @@
+package io.eventuate.examples.tram.ordersandcustomers.commondomain;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+import java.math.BigDecimal;
+
+public class MoneyDTO {
```

Comment:
Remind me again why it's necessary to have Money and MoneyDTO (along with DTO classes such as OrderDetailsDTO). What's the problem with the JPA annotations?

Reply:
Chris,
I don't remember exactly, but there were some dependency conflicts (application did not start).
But as we discussed, I will remove dtos when done with all services.

Reply:
ok. Please remove when done.